TrustModel_Personas.md

```diff
@@ -0,0 +1,197 @@
+# Trust Model Considerations - Personas
+
+## Personas
+Otherwise referred to as actors or threat agents, these are individuals or groups capable of 
```

Comment:
I would have assumed that Trust Model Personas included not only threat agents, but also defense agents.

Reply:
In particular, I think that having a persona owning the attestation servers is lacking in the current version of the document. As written today, it is the "Workload provider", but in practice, I see a very different persona between:

* Someone who wants to create an image for a workload (the Workload provider), essentially a developer, whose role is to ensure that the workload itself follows security best practices, but has little say with respect to how his workload is attested.
* Someone who sets up an attestation server (Attestation provider?) who is in charge of ensuring that the infrastructure running the attestation is safe, etc.

In a sense, it's the same distinction as between container and cloud, but applied to the attestation part. I hope this comment makes sense :-)

Reply:
I don’t see the existing personas as threat agents?
Just agents between which some trust is established?
So each persona can be considered a threat or a defence.

I do agree there are further distinctions we can make among personas but I am conscious of trying to create too many personas too soon. In this case how to introduce the persona relating specifically to the attestation servers for example before we have explained the role of the attestation server? (Chicken and Egg situations everywhere)

The Issue -> https://github.com/confidential-containers/documentation/issues/28 is my attempt to collate at a headline level the content to be reviewed and merged and the next item on this is to explore “Trust Model Artifacts/Components/Services” which will introduce the attestation server among other things.

I guess the open question here is how do we relate to security models which already exist and guide those familiar with existing security models into new personas and new artifacts so they can follow and understand our trust model?

Would you be OK to merge this PR as it is and add to the Issue [#28]  that we need to consider further personas such as around attestation server in further PR’s?

Reply:
I don't object to merging the PR. I agree that this is a bit difficult to put the pieces together, and to find the correct order to do so.

You wrote "I don’t see the existing personas as threat agents?". If that is the case, maybe
```suggestion
Otherwise referred to as actors or agents, these are individuals or groups capable of 
```